CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+2017-10-05 Hoat Le <hoatle@teracy.com>
+--------------------------------------
+
+    Version: 0.14.0
+
+    - Features:
+        * Type hints support for py3 #34
+        * `base_class` introduced #38
+        * Add options passing to `werkzeug.Routing.Rule` from register function #46
+        * Enable using custom decorators in FlaskView #29
+        * Should allow specifying excluded methods from becoming routes #41
+
+    - Improvements:
+        * Better representations #33 #72
+        * Docker workflow along with teracy-dev for better dev setup #32 #63 #66 #67
+
+    - Bugfixes:
+        * The behavior of the trailing slash should be more intuitive and consistent #47
+        * `base_args` should not be overriden when `route_base` is set #50
+
+    - Tasks:
+        * Keep the original license #51
+        * Fix docs publishing #61
+        * Breaking change re: FlaskView.decorators missing in CHANGELOG (for v0.11.0) #49
```

Comment:
Can we make this a larger section? Like a `Breaking Changes` section all on it's own so it's obvious to everyone?

Reply:
It's expected to have some breaking changes before v1.0.0 for faster innovation (following semantic versioning).
I agree, it's great to have a `Breaking Changes` section, maybe you could help with that section content?

Reply:
Totally agreed that we gotta break some stuff to innovate faster.

I'll whip up something really quick here and open a PR.

Apologies for the silence, got sick over the weekend.